docs/topics/whatsnew1530.md

```diff
@@ -32,7 +32,7 @@ and type inference:
 >
 {style="warning"}
 
-An _exhaustive_ [`when`](control-flow.md#when-expression) statement contains branches for all possible types or values of its subject or for some types plus an `else` branch. In other words, it covers all possible cases.
+An _exhaustive_ [`when`](control-flow.md#when-expressions-and-statements) statement contains branches for all possible types or values of its subject or for some types plus an `else` branch. In other words, it covers all possible cases.
```

Comment:
```suggestion
An _exhaustive_ [`when`](control-flow.md#when-expressions-and-statements) statement contains branches for either (a) all possible types or values of its subject, or (b) for some types plus an `else` branch. In other words, it covers all possible cases.
```

This sentence is a bit hard to read. I'm not sure my suggestion is any better, though.

Reply:
It reads better. Still by brains parses "for some types with an `else` branch" as "for some (types with an `else` branch)" instead of "(for some types) with (an `else` branch)".